backend/fms_core/viewsets/workflow.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from django.db.models import Prefetch
+
+from fms_core.models import Workflow, StepOrder
+from fms_core.serializers import WorkflowSerializer
+
+from ._constants import _workflow_filterset_fields
+
+class WorkflowViewSet(viewsets.ModelViewSet):
+    queryset = Workflow.objects.prefetch_related(Prefetch("StepsOrder", queryset=StepOrder.objects.select_related("step"))).all()
+    serializer_class = WorkflowSerializer
+    pagination_class = None
```

Comment:
Setting the pagination_class to None changes the response received by the frontend. In 99% of the api calls, the data is returned as an object with a `results` property, and in the frontend we get that data like this: `action.data.results`. 
If the pagination_class is None, then we just receive an array of workflow objects (in `action.data`).
Was this intentional?

Reply:
yeah and no i did not expect the format to change for the frontend but i thought this was a call we would not want to page. But we can just send a high limit and circumvent the issue. I'll remove it if this make things simpler.

Reply:
I think it's simpler (for me) if all the api's return a consistent style of response.

Reply:
So far we've tried to send back id for foreign keys for the user interface API. The exports generally have more human readable formats.